src/relayserver/RelayServer.ts

```diff
@@ -540,8 +545,8 @@ latestBlock timestamp   | ${latestBlock.timestamp}
 
   async _refreshGasPrice (): Promise<void> {
     const gasPriceString = await this.gasPriceFetcher.getGasPrice()
-    this.gasPrice = Math.floor(parseInt(gasPriceString) * this.config.gasPriceFactor)
-    if (this.gasPrice === 0) {
+    this.minGasPrice = Math.floor(parseInt(gasPriceString) * this.config.gasPriceFactor)
```

Comment:
please add a check here that there is a sane gap between a new `minGasPrice` and a hard-capped `maxGasPriced`.

I think if the difference is small it should emit a warning, if max < min - crash or something, I don't know (it will not be able to relay anything)

Reply:
fixed